### DISCUSSION TITLE
Export (restricted) instance not working

### DISCUSSION THREAD

**Discussion by bijlevel** [RomanKubiak/ctrlr#572] [Q&A] [answered]
I tried to generate an app for my panel, but I get an error message. See attached screenshots. I suspect it has to do with my operating system (Mac OS Monterey) because with El Capitan on another computer it goes OK. I'm using v 5.6.23, because older versions crash immediately. Could it be I have to recompile Ctrlr on Monterey? BTW generating an restricted AU instance from my DAW does succeed...
<img width="281" alt="Screenshot 2023-10-17 at 13 54 07" src="https://github.com/RomanKubiak/ctrlr/assets/11859363/1cc2a3d9-0fd4-4acf-a1f1-7e349060d052">
<img width="484" alt="Screenshot 2023-10-17 at 13 54 20" src="https://github.com/RomanKubiak/ctrlr/assets/11859363/737f74aa-242d-41f2-8652-88e2a05bf2f3">

**Comment by damiensellier** [comment, marked as answer]
I'm still using Catalina so I'm not sure about your problem. 
But It seems that the OS Native open/save file dialog box is not working in newer macOS versions, it could be coming from that.

Can you please try using the JUCE/CTRLR open/save dialog window by unticking the option in the Ctrlr settings?
<img width="739" alt="settings" src="https://github.com/RomanKubiak/ctrlr/assets/55164385/f4ea73d8-36d5-4070-9c86-4b84a4a7ba70">

The embeded file tool looks bad but maybe it will let you save your panel after giving  it the authorisation to navigate your HD.

And let us know if you succeded in exporting your panel ;)
Good luck
Damien

**Reply by bijlevel** [reply]
Yeah, that worked! I think it indeed has to to with more recent MacOS's. On my PowerMac (2008) wit El Capitan the export worked. That was the detour I had to take. Odd though, exporting as an AU from my DAW (Digital Performer) with MacOS Monterey did succeed. Anyway, thanks a lot Damien!
